app/api/dao/user.py

```diff
@@ -128,8 +131,14 @@ def update_user_profile(user_id, data):
         if 'resume_url' in data and data['resume_url']:
             user.resume_url = data['resume_url']
 
-        if 'photo_url' in data and data['photo_url']:
-            user.photo_url = data['photo_url']
+        try:
+            if 'photo_url' in data and data['photo_url']:
+                image_type = data['photo_url'].split(';')[0].split('/')[1]
+                image_name = username + "." + image_type
```

Comment:
join is better here.

Reply:
image_name = ".".join((username, image_type)) How about this.